modules/tinymce/src/themes/silver/main/ts/ui/toolbar/button/ToolbarButtons.ts

```diff
@@ -303,140 +304,183 @@ const fetchChoices = (getApi: (comp: AlloyComponent) => Toolbar.ToolbarSplitButt
         )
       )));
 
-// TODO: hookup onSetup and onDestroy
-const renderSplitButton = (spec: Toolbar.ToolbarSplitButton, sharedBackstage: UiFactoryBackstageShared, btnName?: string): SketchSpec => {
-  const tooltipString = Cell<string>(spec.tooltip.getOr(''));
+const makeSplitButtonApi = (tooltipString: Cell<string>, sharedBackstage: UiFactoryBackstageShared) => (component: AlloyComponent): Toolbar.ToolbarSplitButtonInstanceApi => {
+  const system = component.getSystem();
+  const element = component.element;
+  const isChevron = Class.has(element, 'tox-split-button__chevron');
+
+  const mainOpt = isChevron ?
+    Traverse.prevSibling(element).bind((el) => system.getByDom(el).toOptional()) :
+    Optional.some(component);
+
+  const chevronOpt = isChevron ?
+    Optional.some(component) :
+    Traverse.nextSibling(element).bind((el) => system.getByDom(el).toOptional().filter((comp) => Class.has(comp.element, 'tox-split-button__chevron')));
+
```

Comment:
I think we try to avoid using DOM positions as such to identify related components 🤔 
Should we use `Memento` instead, or maybe use an internal alloy event to tell them both to update the tooltips?